content/en/docs/instrumentation/swift/manual.md

```diff
@@ -0,0 +1,155 @@
+
+```swift
+let span =  let builder = tracer.spanBuilder(spanName: "\(name)").startSpan()
+...
+span.end()
+```
+It is required to call `end()` to end the span.
+
+### Creating Nested Spans
+
+Often, spans are used to build relationship between operations. 
+Below is an example of how we can manually build relationship between spans.
+
+Below we have `parent()` calling `child()` and how to manually link spans of each of these methods.
+
+```swift
+func parent() {
+  let parentSpan = someTracer.spanBuilder(spanName: "parent span").startSpan() 
+  child(span: parentSpan) 
+  parentSpan.end()
+}
+
+func child(parentSpan: Span) {
+  let childSpan = someTracer.spanBuilder(spanName: "child span")
+                             .setParent(parentSpan)
+                             .startSpan()
+  // do work
+  childSpan.end()
+}
+
+```
+
+The parent-child relationship will be automatically linked if `activeSpan` is used:
+
+```swift
+func parent() {
+  let parentSpan = someTracer.spanBuilder(spanName: "parent span")
+                      .setActive(true) // automatically sets context
+                      .startSpan() 
+  child() 
+  parentSpan.end()
+}
+
+func child() {
+  let childSpan = someTracer.spanBuilder(spanName: "child span")
+                             .startSpan() //automatically captures `active span` as parent
+  // do work
+  childSpan.end()
+}
+
+```
+
+### Getting the Current Span
+
+Sometimes it's useful to do something with the current/active span. Here's how to access the current span from an arbitrary point in your code.
+```swift 
+  let currentSpan = OpenTelemetry.instance.contextProvider.activeSpan
+```
+### Span Attributes
+
+Spans can also be annotated with additional attributes. All spans will be automatically annotated with the `Resource` attributes attached to the tracer provider.
+The Opentelementry-swift sdk already provides instrumentation of common attributes in the `SDKResourceExtension` instrumentation.
+In this example a span for a network request capturing details about that request using existing [semantic conventions](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/).
+```swift
+let span = tracer.spanBuilder("/resource/path").startSpan()
+span.setAttribute("http.method", "GET");
+span.setAttribute("http.url", url.toString());
+```
+
+### Creating Span Events
+A Span Event can be thought of as a structured log message (or annotation) on a Span, typically used to denote a meaningful, singular point in time during the Span’s duration.
+
+```swift
+            let attributes = [
+                "key" : AttributeValue.string("value"),
+                "result" : AttributeValue.int(100)
+            ]
+            span.addEvent(name: "computation complete", attributes: attributes)
+```
+
+### Setting Span Status
+  
+### Recording Errors in Spans
+
+## Acquiring A Meter
+
+
```

Comment:
```suggestion
```

We can add those additional sections also in a follow-up PR, no problem in building this gradually.